src/extent.c

```diff
@@ -902,16 +915,14 @@ extent_recycle_extract(tsdn_t *tsdn, arena_t *arena,
 			}
 			extent_unlock(tsdn, unlock_extent);
 		}
+		if (extent) {
+			extent_activate_locked(tsdn, arena, extents, extent);
```

Comment:
Is this a no-op? It seems like everywhere we *get* an extent from ought to be activating it for us.

Reply:
No, we get it by addr here, the one place it isn't already activated for us.